packages/mobile/src/components/feed-filter-drawer/FeedFilterDrawer.tsx

```diff
@@ -74,7 +61,11 @@ export const FeedFilterDrawer = () => {
   return (
     <ActionDrawer
       modalName={MODAL_NAME}
-      renderTitle={() => <Text style={styles.title}>{messages.title}</Text>}
+      title={
+        <Text color='neutral' textTransform='none'>
```

Comment:
this is about to get wrapped in another text inside DrawerHeader, right?

Reply:
should we take a different prop when you are passing a component vs plain text?

Reply:
i could def see that, i was thinking if you want to update the title text, you can just pass a text component ready to go, and it's okay that it's wrapped. really only applies to one drawer and ideally we should all have the same title. can see the idea of having a "renderTitle" thing that goes all the way down though.

Reply:
yeah either way is okay I think. As long as double wrapping the text uses the inner one I guess